src/Integrator/Hydro.cpp

```diff
@@ -233,19 +236,6 @@ void Hydro::Initialize(int lev)
 
     Source_mf[lev]   ->setVal(0.0);
 
```

Comment:
This initialization function assumes that all IO has already taken place - "gas" should exist at this point.